modules/microshift-manifests-removal-scenario-ostree.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-removal-scenario-ostree_{context}"]
+== Manifest removal scenario for OSTree systems
+Use the following procedure in the data removal scenario for OSTree systems to completely delete the resource defined in the kustomize manifests.
+
+[IMPORTANT]
+====
+For OSTree installation, the `delete` subdirectories are read only.
+====
+
+.Procedure
+
+. Identify the manifest that needs to be placed in the `delete` subdirectories.
+. Package the manifest into an RPM. See link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/microshift-embedding-apps-tutorial#microshift-building-apps-rpms_microshift-embedding-apps-tutorial[Building the RPM package for the application] for the procedure to package the manifest into an RPM.
```

Comment:
Hi, edit to my previous comment - I'm told it's fine in Microshift to link directly to the content (instead of what I suggested with xrefs), but the links should be to the location on docs.redhat.com since that has replaced "access.redhat.com". Hope this helps!

Reply:
done.